### DISCUSSION TITLE
How to import CSV into Excel without hash becoming formulas

### DISCUSSION THREAD

**Discussion by sparkmedia** [davetapley/helium-tax#53] [General]
As the hash column may occasionally start with a - symbol, Excel will interpret this as a formula and give you #NAME? as the result.  To avoid this...

1) From Excel choose Data -> Get Data -> From Text.  This will force excel to use the TEXT Import Wizard.
2) Choose Delimited, disable Tab and enable Comma.
3) On the third screen, be sure to select the last column (hash) and change the Column Data Format from General to Text.

You're welcome.

**Comment by davetapley** [comment]
Thanks 🙏🏻
